invites/async/reply.js

```diff
@@ -27,14 +27,13 @@ module.exports = function (server) {
 
     getInvite(reply.branch, (err, invite) => {
```

Comment:
:fire: I don't think branch should be used like this.
This presumes `params.branch` exists, where I think one of the params or args passed in should be the invite key you're replying to.

like `invite.async.reply(inviteKey, { body: 'yep' reply: true }, cb` or something,

Reply:
the reason I say this is the in the ecosystem `branch` can be an Array sometimes, which would fuck up this method.
I think I'll raise this as a seperate issue though

Reply:
:white_check_mark: Done